generators/node/index.js

```diff
@@ -537,10 +554,22 @@ module.exports = class extends BaseGenerator {
 
 		const context = this.appConfig
 		const path = context.name
-		this.fs.copyTpl(this.sourceRoot() + '/index.js', path + '/index.js', context)
 		this.fs.copyTpl(this.sourceRoot() + '/package.json', path + '/package.json', context)
 		this.fs.copyTpl(this.sourceRoot() + '/README.md', path + '/README.md', context)
 
+		this.fs.copyTpl(
+			this.templatePath(this.sourceRoot() + '/' + this.appConfig.stSchemaTemplate),
+			this.destinationPath('./' + path),
+			{title: 'Overwriting template'}
+		)
+
+		// If user select template
+		this.fs.copy(
+			this.templatePath(this.sourceRoot() + '/' + this.appConfig.stSchemaTemplate),
+			this.destinationPath('./' + path),
+			{title: 'Overwriting template'}
+		)
```

Comment:
we just need copy?? if that I'll fix next PR

Reply:
Sorry, I forgot to comment on this. Yeah, you should only need to do `copy` or `copyTpl` but not both. If the file you are copying is a template file (or you think it might include template stuff at some point, use `copyTpl`. If you want it to just copy without any template substitutions, use `copy`.